clang/lib/DPCT/RulesAsm/Parser/AsmNodes.h

```diff
@@ -83,6 +83,7 @@ class InlineAsmBuiltinType : public InlineAsmType {
   enum TypeKind {
 #define BUILTIN_TYPE(X, Y) X,
 #include "AsmTokenKinds.def"
+    voidType,
```

Comment:
voidType is not needed, pls removed.

Reply:
Removed it.

Using **byte** type now as per your suggestions.